src/software/field_tests/field_test_fixture.py

```diff
@@ -320,6 +320,27 @@ def load_command_line_arguments():
         help="Run the test with friendly robots in yellow mode",
     )
 
+    parser.add_argument(
+        "--sslvision_multicast_address",
+        type=str,
+        default=SSL_VISION_ADDRESS,
+        help="the multicast address for ssl vision",
+    )
+
```

Comment:
we don't need this arg, we can default to SSL_VISION_ADDRESS and just change the port